certified-connectors/OpenExperience/apiDefinition.swagger.json

```diff
@@ -0,0 +1,249 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Open Experience",
+    "description": "This is an Open Experience connector, which enables you to integrate the Open Experience services with your flows",
```

Comment:
@oxTomaszJ The given description is not sufficient, please provide a detailed description of your service and connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)